### ISSUE TITLE
opcua-commander closes immediately after connecting

### ISSUE THREAD

**Issue by jdion-tft** [node-opcua/opcua-commander#13]
I am able to connect to the opcua server with commander, but immediately after connecting the program disconnects from the server, and closes the output in the terminal window. No error message is given.

I'm running the latest node-opcua and opcua-commander code.

**Comment by roykn** [comment]
Try to set the option **endpoint_must_exist** in index.js:84 to **false**, like:
```javascript
const options = {
    securityMode: securityMode,
    securityPolicy: securityPolicy,
    //xx serverCertificate: serverCertificate,
    defaultSecureTokenLifetime: 40000,
    certificateFile: certificateFile,
    privateKeyFile: privateKeyFile,
    endpoint_must_exist: false
};
```
This works for me.

**Comment by ckarras** [comment]
endpoint_must_exist doesn't work for me. How can I get an error message for more investigation?

**Comment by sachinmorabada** [comment]
Am also facing same problem opcua-commander closes immediately after connecting. Even after updating     endpoint_must_exist: false.

**Comment by qlevZer** [comment]
I have the same probleme...

**Comment by alexshn** [comment]
@qlevZer Try to remove [this line](https://github.com/node-opcua/opcua-commander/blob/master/index.js#L192). It forces commander to close immediatelly and you are not able to see the message.

**Comment by sanderd17** [comment]
I also had this, and it's related to certificates.

I when I connected to the device using UA-Expert, it asked to install a certificate. After connecting, I saw the PLC was used with this endpoint: `opc.tcp://Grading:4840`

Later, when connecting to the PLC using opcua-commander, I could connect via `opc.tcp://Grading:4840`, but not via `opc.tcp://GRADING:4840` (although hostnames are case insensitive in windows, and I can ping both), and going via the ip address `opc.tcp://172.16.13.120:4840` also didn't work.

I'm not sure how UA-Expert detects a lacking certification, and even generates one.

Btw, removing that line did help to see the message.

![image](https://user-images.githubusercontent.com/562171/48259437-338c9780-e418-11e8-80ad-c3baafd854dc.png)
![image](https://user-images.githubusercontent.com/562171/48259412-253e7b80-e418-11e8-8810-82f1e7a0586e.png)

**Comment by erossignon** [comment]
* A small delay have been added before closing the windows so user can read the error message
* endpoint_must_exist = false  has been set to allow endpointUrl that do not match endpoint url specified in the server certificate.  
-> available in opcua-commander@0.3.0
